lib/assesments/assessment_2/fi-as2-hira/Screens/Login/widgets/fi_as2_hira_circlecontainer.dart

```diff
@@ -0,0 +1,25 @@
+part of '../fi_as2_hira_login.dart';
+
+class H1Circle extends StatelessWidget {
+  final Widget child;
+  const H1Circle({super.key, required this.child});
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      height: 50,
+      width: 50,
+      decoration: BoxDecoration(
+          color: Constants.grey, borderRadius: BorderRadius.circular(50),),
+      padding: const EdgeInsets.all(3.0),
+      child: ClipRRect(
```

Comment:
Why use another widget, you can provide borderRadius directly to Container in decoration
```
decoration: BoxDecoration(
borderRadius: and so on,
)
```